test/testdata/namer/type_member_redefs.rb

```diff
@@ -0,0 +1,20 @@
+# typed: true
+# disable-fast-path: true
```

Comment:
why?

Reply:
Pending the `show` fix I'd like to land today. (This has a `R$1` that shows up in the fast path which breaks the error checks.)

Reply:
Can we add link to the issue blocking this to the file?

Reply:
I might as well just fix it: https://github.com/sorbet/sorbet/pull/1787